openff/toolkit/topology/molecule.py

```diff
@@ -2792,6 +2793,62 @@ def generate_conformers(
                 )
             )
 
+    def apply_elf_conformer_selection(
+        self,
+        percentage: float = 2.0,
+        limit: int = 10,
+        toolkit_registry: Optional[
+            Union[ToolkitRegistry, ToolkitWrapper]
+        ] = GLOBAL_TOOLKIT_REGISTRY,
+        **kwargs,
+    ):
+        """Applies the `ELF method<https://docs.eyesopen.com/toolkits/python/quacpactk/
+        molchargetheory.html#elf-conformer-selection>`_ to select a set of diverse
+        conformers which have minimal electrostatically strongly interacting functional
+        groups from a molecules conformers.
+
+        Notes
+        -----
+        * The input molecule should have a large set of conformers already
+          generated to select the ELF conformers from.
+        * The selected conformers will be retained in the `conformers` list
+          while unselected conformers will be discarded.
+
+        See Also
+        --------
+        ``OpenEyeToolkitWrapper.apply_elf_conformer_selection``
+        ``RDKitToolkitWrapper.apply_elf_conformer_selection``
+
+        Parameters
+        ----------
+        toolkit_registry
+            The underlying toolkit to use to select the ELF conformers.
+        percentage
+            The percentage of conformers with the lowest electrostatic interaction
+            energies to greedily select from.
```

Comment:
(not blocking) Will there be an API change if we switch to the OE-style "actually use a higher percentage if there aren't many input conformers" approach? Or would we make a separate method for that? We may want to plan ahead so that we end up with a clear API.

Reply:
I think the API would remain the same, just a docstring update and a note about the behaviour change during a release would be fine.